tox.ini

```diff
@@ -146,7 +146,8 @@ commands =
     black: black {env:BLACK_LINT_ARGS} {posargs:{[default]sources}}
 
     apidocs: {toxinidir}/bin/admin/build-apidocs {toxinidir}/src/ apidocs
-    narrativedocs: sphinx-build -aW -b html -d {toxinidir}/docs/_build {toxinidir}/docs {toxinidir}/docs/_build/
+    narrativedocs: sphinx-build -aW -W --keep-going -b html -d {toxinidir}/docs/_build {toxinidir}/docs {toxinidir}/docs/_build/
+    narrativedocs: sphinx-build -aW -W --keep-going -b epub -d {toxinidir}/docs/_build/doctrees-epub {toxinidir}/docs {toxinidir}/docs/_build/epub
```

Comment:
just a comment. Note that in twisted we have twisted.python._release.SphinxBuilder and bin/admin/build-docs which are the tools designed to be used to release the documentation.

I think that as part of the automated continuous tests, we should build the documentation using those tools.... or if we found that those tools are no longer needed, remove them.

But the idea is that those tools are useful for a release, we should run the tests using those tools.

This is what is already done for pydoctor API docs, where we don't call pydoctor directly, but use the bin/admin/build-apidocs herlper